integrations/utils.ts

```diff
@@ -177,12 +147,14 @@ export function test(
 
           child.stdout.on('data', (result) => {
             let content = result.toString()
+            if (debug) console.log(content)
```

Comment:
Should this have like a `[stdout] ` prefix or something? Just so it's more easily distinguishable in the test output?

Reply:
thinking out loud: should probably prefix every line of the output

Reply:
Hmm, maybe? It could help with the `onStderr` helper we have but I wonder why we need to be so diligent about this in the first place. In normal terminal apps you don't see the difference between `stdout` and `stderr` in the console either unless you pipe line stream to another location. I decided to replicate this here by using `console.log` for `stdout` and `console.error` for `stderr`

Reply:
I personally don't think the `[stdout]` or `[stderr]` prefix helps here because Vitest already shows that for us:

<img width="625" alt="image" src="https://github.com/user-attachments/assets/216c5397-bbe3-478e-add3-d0e4e926c4e1">